sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/BuilderHelper.java

```diff
@@ -140,15 +145,24 @@ private static void validateSingleCredentialIsPresent(
      * Creates a {@link UserAgentPolicy} using the default blob module name and version.
      *
      * @param configuration Configuration store used to determine whether telemetry information should be included.
+     * @param clientOptions ClientOptions such as application ID and custom headers to set on a request.
+     * @param logOptions The logging configuration to use when sending and receiving requests to and from the service.
      * @return The default {@link UserAgentPolicy} for the module.
      */
-    private static UserAgentPolicy getUserAgentPolicy(Configuration configuration) {
-        configuration = (configuration == null) ? Configuration.NONE : configuration;
+    private static UserAgentPolicy getUserAgentPolicy(Configuration configuration, ClientOptions clientOptions,
+        HttpLogOptions logOptions) {
 
         String clientName = PROPERTIES.getOrDefault(SDK_NAME, "UnknownName");
```

Comment:
Should make `clientName` and `clientVersion` constants, they won't change during runtime